framework_tvb/tvb/core/neocom/h5.py

```diff
@@ -309,3 +313,86 @@ def gather_all_references_of_view_model(gid, base_dir, ref_files):
             uuid_files.append(h5_file.path)
             gather_all_references_by_index(h5_file, uuid_files)
         ref_files.extend(uuid_files)
+
+
+# Dependency dict
+h5_files_dependencies = {
+    "RegionVolumeMapping": ["Volume"],
+    "StructuralMRI": ["Volume"],
+    "TimeSeriesVolume": ["Volume"]
```

Comment:
This is not nice and Could easily become outdated. Can't there be an automatic way?
At import time for example we order by create-date and that proved enough.